api/SpeechSynthesisEvent.json

```diff
@@ -201,37 +201,53 @@
           "spec_url": "https://wicg.github.io/speech-api/#dom-speechsynthesisevent-elapsedtime",
           "support": {
             "chrome": {
-              "version_added": "33"
+              "version_added": "33",
+              "notes": "Units are miliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
             },
             "chrome_android": {
-              "version_added": "33"
+              "version_added": "33",
+              "notes": "Units are miliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
             },
             "edge": {
-              "version_added": "14"
-            },
-            "firefox": {
-              "version_added": "49"
-            },
+              "version_added": "14",
+              "notes": "Units are miliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
```

Comment:
I'm aware of the fact the underlying engine changed, but this should be correct until chromium fixes the bug.